doc/r1-design.md

```diff
@@ -0,0 +1,148 @@
+a placeholder to denote a connectivity between an instance's attachment point to another instance's attachment point
+
+As part of monitoring the deployment once the packages are deployed, NTC will also create a NFDeployedTopology resource on the management cluster to trigger monitoring from EdgeStatus Aggregator (ESA). The following denotes the CRD representing the input for ESA:
+
+```golang
+type NFDeployedConnectivity struct {
+    // peer NF's Id (see NFInstance struct below)
+	NeighborName string `json:"neighborName,omitempty" yaml:"neighborName,omitempty"`
+}
+
+type NFDeployedInstance struct {
+    // unique ID for this NF instance
+	Id             string         `json:"id,omitempty" yaml:"id,omitempty"`
+    // name of workload cluster where the NF instance is to be deployed
+	ClusterName    string         `json:"clustername,omitempty" yaml:"clustername,omitempty"`
+    // type of NF, example: amf, smf, upf
+	NFType         string         `json:"nftype,omitempty" yaml:"nftype,omitempty"`
+    // flavor of NF, example: small, medium, large
+	NFFlavor       string         `json:"nfflavor,omitempty" yaml:"nfflavor,omitempty"`
+    // NF vendor name
+	NFVendor       string         `json:"nfvendor,omitempty" yaml:"nfvendor,omitempty"`
+    // the software version of this NF vendor's NFType
+	NFVersion      string         `json:"nfversion,omitempty" yaml:"nfversion,omitempty"`
+    // list of connected NF instances to this NF instance
+	Connectivities []NFDeployedConnectivity `json:"connectivities,omitempty" yaml:"connectivities,omitempty"`
+}
+
+// NFDeployedTopologySpec defines the desired state of NfToplogy deployment
+type NFDeployedTopologySpec struct {
+	NFInstances     []NFDeployedInstance `json:"nfinstances,omitempty" yaml:"nfinstances,omitempty"`
+}
+```
+
+i.e., this serves as a simplfied version of NFTopology above and making it simpler for ESA to process more in deployment tracking purposes; the tradeoff for simplicity is that there is repeated information on each instance's Connectivities slice
+
+## NFTopology to PackageVariantSet CR Mapping
+NTC kickstarts the hydration process via applying the **PackageVariantSet** (PVS) CR to management cluster. The following table denotes which fields from NFTopology (and its associated struct) will be used to populate the PVS CR:
+
+| NFTopology or associated | PVS Spec field | 
+|:------------------------ | :------------- |
+| NFTopology.NFInstance.NFTemplate.ClassRef.PackageRevisionRef | Upstream (*Tag* is missing from PackageRevisionRef... Revision?) | 
+| NFTopology.NFInstance.ClusterSelector | Targets[0].Repositories |
+| {nf-deployment-name : *name of NFTopology CR* | Labels[] | 
+
+The assumption here is that (at least a subset of) the ClusterSelector labels will be used to also label the repositories, i.e., target workload cluster with label `region == x; cluster-role == y` will also be part of the labels to use on a deployment repo of a target workload cluster.
```

Comment:
I would suggest labeling it with a label based on this NFTopology resource name, so that you can aggregate it specifically for this NFTopology. You also add the NFTopology to the ownerReferences on the PVSs. But the labels (e.g., `nf.nephio.org/topology` or something) can be used in watches and can, like you say, be propagated all the way down to the PackageRevisions.